export const data = JSON.parse("{\"key\":\"v-576c0984\",\"path\":\"/go-advancend/markdown/31.html\",\"title\":\"第31节 程序中的错误处理\",\"lang\":\"zh-CN\",\"frontmatter\":{},\"excerpt\":\"\",\"headers\":[{\"level\":3,\"title\":\"Go语言基础篇\",\"slug\":\"go语言基础篇\",\"link\":\"#go语言基础篇\",\"children\":[]},{\"level\":3,\"title\":\"Go语言100篇进阶\",\"slug\":\"go语言100篇进阶\",\"link\":\"#go语言100篇进阶\",\"children\":[]},{\"level\":2,\"title\":\"前言\",\"slug\":\"前言\",\"link\":\"#前言\",\"children\":[]},{\"level\":2,\"title\":\"多返回值\",\"slug\":\"多返回值\",\"link\":\"#多返回值\",\"children\":[]},{\"level\":2,\"title\":\"资源清理\",\"slug\":\"资源清理\",\"link\":\"#资源清理\",\"children\":[]},{\"level\":2,\"title\":\"异常捕捉处理\",\"slug\":\"异常捕捉处理\",\"link\":\"#异常捕捉处理\",\"children\":[]},{\"level\":2,\"title\":\"错误返回码 vs 异常捕捉\",\"slug\":\"错误返回码-vs-异常捕捉\",\"link\":\"#错误返回码-vs-异常捕捉\",\"children\":[]},{\"level\":2,\"title\":\"异步编程世界里的错误处理\",\"slug\":\"异步编程世界里的错误处理\",\"link\":\"#异步编程世界里的错误处理\",\"children\":[]},{\"level\":2,\"title\":\"Go 语言的 Promise\",\"slug\":\"go-语言的-promise\",\"link\":\"#go-语言的-promise\",\"children\":[]},{\"level\":2,\"title\":\"错误处理的实践\",\"slug\":\"错误处理的实践\",\"link\":\"#错误处理的实践\",\"children\":[]},{\"level\":2,\"title\":\"END 链接\",\"slug\":\"end-链接\",\"link\":\"#end-链接\",\"children\":[]}],\"git\":{\"updatedTime\":1676737288000,\"contributors\":[{\"name\":\"xiongxinwei\",\"email\":\"3293172751nss@gmail.com\",\"commits\":3},{\"name\":\"Xinwei Xiong\",\"email\":\"3293172751nss@gmail.com\",\"commits\":1}]},\"filePathRelative\":\"go-advancend/markdown/31.md\"}")
